app/components/groups/group-edit/edit-group.controller.js

```diff
@@ -4,8 +4,36 @@
 
     .controller("EditGroupController", EditGroupController)
 
-    EditGroupController.$inject = ["$scope"]
+    EditGroupController.$inject = ["GroupResource", "$modal", "$scope",
+      "$state", "$stateParams"]
 
-    function EditGroupController($scope){
+    function EditGroupController(GroupResource, $modal, $scope, $state, $stateParams){
```

Comment:
in this part I don't use $state, but I use it here https://github.com/NikitaSedyx/todo-list/pull/24